src/code42cli/cmds/alert_rules.py

```diff
@@ -151,24 +149,14 @@ def handle_row(rule_id, username):
 
 def _add_user(sdk, rule_id, username):
     user_id = get_user_id(sdk, username)
-    rules = _get_rule_metadata(sdk, rule_id)
-    try:
-        if rules:
-            sdk.alerts.rules.add_user(rule_id, user_id)
-    except Py42InternalServerError:
-        _check_if_system_rule(rules)
-        raise
+    _get_rule_metadata(sdk, rule_id)  # Verifies rule exists
+    sdk.alerts.rules.add_user(rule_id, user_id)
 
 
 def _remove_user(sdk, rule_id, username):
     user_id = get_user_id(sdk, username)
-    rules = _get_rule_metadata(sdk, rule_id)
-    try:
-        if rules:
```

Comment:
Perhaps I should leave this part `if rules`

Reply:
if this can return an empty list rather than erroring, then yes